packages/features/auth/lib/next-auth-options.ts

```diff
@@ -844,7 +845,7 @@ export const AUTH_OPTIONS: AuthOptions = {
               where: { email: existingUserWithEmail.email },
               // also update email to the IdP email
               data: {
-                email: user.email,
+                email: user.email.toLowerCase(),
```

Comment:
We convert to lower case to ensure an uppercase email doesn't slip in from the idP into our DB